pkg/microservice/aslan/core/project/handler/service_policy.yaml

```diff
@@ -72,6 +74,8 @@ rules:
         endpoint: "/api/aslan/build/build/?*"
       - method: GET
         endpoint: "/api/aslan/build/targets"
+      - method: GET
+        endpoint: "api/aslan/cluster/clusters"
```

Comment:
What's the difference between `api/aslan/cluster/clusters` and `/api/aslan/cluster/clusters"`?

Reply:
no difference